app/models/miq_provision/naming.rb

```diff
@@ -23,7 +23,7 @@ def get_next_vm_name(prov_obj, determine_index = true)
         options        = prov_obj.options
         options[:tags] = prov_obj.get_tags
 
-        load File.join(File.expand_path(Rails.root), 'db/fixtures/miq_provision_naming.rb')
+        load File.join(ApplicationRecord::FIXTURE_DIR, "miq_provision_naming.rb")
```

Comment:
Tried that but it broke :confused: It's a Concern, so only using `FIXTURE_DIR` references `MiqProvision::Naming::FIXTURE_DIR`